nvtabular/dask/io.py

```diff
@@ -197,9 +196,10 @@ def __init__(
         # If engine is not provided, try to infer from end of paths[0]
         if engine is None:
             engine = paths[0].split(".")[-1]
-
         if isinstance(engine, str):
             if engine == "parquet":
+                if "names" in kwargs:
+                    kwargs.pop("names")
```

Comment:
Do we need to pop off names here - is it even used anywhere? Like can we just remove setting the 'names' attribute in the unittest on test_io.py line 63 instead?